### PR TITLE
Add jenkins views states

### PR DESCRIPTION
**Pull request by toopy**

### CODE REVIEW

**Comment by toopy** [comment]
@bersace opm ?

File: jenkins/views/uninstall.sls [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  jenkins_views.absent:
+    - names:
+{% for name in present %}
+      - {{ name }}
+{% endfor -%}
+{% endif %}
+
+{%- if views_jobs_present %}
+{%- for view, jobs in views_jobs_present.iteritems() %}
+views_jobs_absent_{{ view }}:
```
Comment (bersace) [inline jenkins/views/uninstall.sls]: Why delete each job on absent view ?

File: jenkins/views/uninstall.sls [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  jenkins_views.job_absent:
+    - names:
+{%- for job in jobs %}
+      - {{ job }}
+{%- endfor %}
+    - view: {{ view }}
+{%- endfor %}
+{%- endif %}
+
+restart_after_views_uninstall:
```
Comment (bersace) [inline jenkins/views/uninstall.sls]: Is reload enough for views ?

**Comment by toopy** [comment]
@bersace up'

File: _states/jenkins_views.py [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+        ret['changes'][view] = {
+            'old': old,
+            'new': new,
+        }
+
+    ret['result'] = None if __opts__['test'] else True  # noqa
+    return ret
+
+
+def job_present(name, names=None, view=None, **kwargs):
```
Comment (bersace) [inline _states/jenkins_views.py:99]: Why `view=None` if required ?

File: _states/jenkins_views.py [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+    })
+
+    # update
+    if not __opts__['test']:  # noqa
+        try:
+            _runcli(command, view, input_=new)
+        except exc.CommandExecutionError as e:
+            ret['comment'] = e.message
+            return ret
+    else:
```
Comment (bersace) [inline _states/jenkins_views.py:151]: Useless :)

Comment (toopy) [inline _states/jenkins_views.py:151]: What else ?

**Pull request merged**